js/pulse.js

```diff
@@ -916,15 +965,17 @@
                 }
             };
 
-            var _onAdFinished = function(){
+            var _onAdFinished = function() {
                 amc.notifyLinearAdEnded(1);
+                enableAdScreenPointerEvents();
             };
 
-            var _onAdSkipped = function(){
+            var _onAdSkipped = function() {
                 amc.notifyLinearAdEnded(1);
+                enableAdScreenPointerEvents();
```

Comment:
I've added it, but without `notifyLinearAdEnded`.
Not every `LINEAR_AD_ERROR` has a corresponding `LINEAR_AD_START` so I'm not sure what the implications might be of calling `notifyLinearAdEnded` too often.

Since it's worked so far, I decided to just leave it out until we can look into it more.